mantle/rbx_mantle/src/config.rs

```diff
@@ -523,6 +523,20 @@ pub struct ExperienceTargetConfig {
 
     /// Spatial voice configuration.
     pub spatial_voice: Option<SpatialVoiceTargetConfig>,
+
+    /// Notification strings configuration.
+    pub notifications: Option<Vec<NotificationTargetConfig>>,

```

Comment:
Let's make this a `HashMap` so that it's more obvious for the creator to give it a more machine-readable name so that they can programmatically reference the resulting notification ID from the Mantle outputs file.